spec/requests/follow_relationships_spec.rb

```diff
@@ -0,0 +1,194 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'FollowRelationships', type: :request do
+  describe 'POST /create' do
```

Comment:
Let's add one test here to see what happens if you try to follow a user you already follow (count shouldn't increase, button should be changed). Come here after changing my previous comment in `app/models/follow_relationship.rb`